taccsite_cms/static/site_cms/css/src/_imports/trumps/s-header.css

```diff
@@ -34,8 +32,10 @@ Styleguide Trumps.Scopes.Header
   /* Copied from Portal (via `body`) */
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
-}
 
+  /* To hide border, set this in theme to match `--header-bkgd-color` value */
+  border-bottom: 1px solid env(--header-major-border-color);
```

Comment:
**Why?** The `has-dark-logo` theme adds a border. The default theme does not have one.